src/ServerTelemetryChannel/SamplingTelemetryProcessor.cs

```diff
@@ -160,7 +187,7 @@ public void Process(ITelemetry item)
             var samplingSupportingTelemetry = item as ISupportSampling;
```

Comment:
I think the condition which checks samplingPercentage >= 100 should honor Next/UnsampledNext.

Reply:
Without it different stream will be affecting and causing oscillation...

Reply:
And we can see this is happening in "after fix" picture (love pictures! =))

Reply:
I made the change in a way that keeps the original intent of the `SamplingTelemetryProcessor` having an early exit for items when the sampling rate is at 100%, but forces evaluations to occur when it's being used as part of the `AdaptiveSamplingTelemetryProcessor` chain.

Same test program and time range as the original 2 charts, but notice there is no lag in events immediately following the initial sampling:

![fixappliedandupdated-dataafterninitialsampleperiod](https://user-images.githubusercontent.com/11466735/33502284-54deebbe-d694-11e7-8242-7d5b99e54582.png)

Reply:
Looks great!